[monitoring] Remove monitoring content from repository

BenjaminPelletier opened the pull request:
This PR completes the [split](https://lists.interussplatform.org/g/tsc/topic/splitting_out_monitoring/94745227) of the [`monitoring` repo](https://github.com/interuss/monitoring) from this repo.  Primarily, all the content of the monitoring folder is removed, except that any .md files which may have external links pointing to them are retained with content replaced with a note pointing to the monitoring repository.  Documentation is also updated to strip the monitoring portions out.

To maintain the existing level of CI for the DSS implementation, probe_locally.sh is added to build/dev (replacing monitoring/prober/run_locally.sh) using an explicit monitoring image tag (interuss/monitoring:v0.1.0) instead of the locally-built monitoring image.  This will cause the public monitoring image to be pulled and used for the CI of this repository.  ~~**Note that CI tests will not pass until the monitoring image is published successfully, which requires merging of [this monitoring PR](https://github.com/interuss/monitoring/pull/7) and a subsequent monitoring release.**~~ (edit: PR merged, monitoring image published)  However, I have tested the CI to succeed locally when provided with a locally-built `interuss/monitoring:v0.1.0` image.

barroco reviewed (commented):
Few findings:
1. There is a reference to `../../monitoring/prober` in dss/build/dev/standalone_instance.md:153
2. There is a reference to `../../monitoring/prober` in test/README.md:34
3. Do we want to clean the assets directory as well?
4. The image used in dss/test/migrations/rid_db_post_migration_e2e.sh is not pinpointed to a specific version of `interuss/monitoring` image. (lines 81 and 88)

barroco reviewed (approved):
See my comments before merging please.

BenjaminPelletier commented:
> There is a reference to ../../monitoring/prober in dss/build/dev/standalone_instance.md:153

Thanks, fixed

> There is a reference to ../../monitoring/prober in test/README.md:34

Thanks, fixed

> Do we want to clean the assets directory as well?

Probably yes.  I didn't do it initially because it takes longer to figure out what to keep and what to delete, but it does seem to make sense to do everything at once for clarity.

> The image used in dss/test/migrations/rid_db_post_migration_e2e.sh is not pinpointed to a specific version of interuss/monitoring image. (lines 81 and 88)

Thanks; I think we'll want to get rid of this file at some point, but fixed for now.
